test/data.jl

```diff
@@ -262,13 +262,13 @@ module TestData
 
     # # TODO: Restore this functionality
     # m1 = join(dt1, dt2, on = [:a,:b])
-    # @test isequal(m1[:a], Vector(["x", "x", "y", "y", fill("x", 5)]))
+    # @test m1[:a] == Vector(["x", "x", "y", "y", fill("x", 5)]))
```

Comment:
OK, I thought I had introduced this when porting to `Nullable`. So we don't really need to tackle it in this PR. It would be great if you could open a PR to either remove these (if that's already covered) or modify them as suitable.